content/docs/setup/kubernetes/helm-install/index.md

```diff
@@ -43,15 +43,13 @@ the appropriate snapshot.  For example, if you want to run with snapshot 6, use
 1.  Update Helm's local package cache with the location of the Helm daily release:
 
     {{< text bash >}}
-
     $ helm repo add istio.io "https://gcsweb.istio.io/gcs/istio-prerelease/daily-build/release-1.1-latest-daily/charts/"
-
```

Comment:
I'm concerned with all these pink lines with a -. What has changed on these lines throughout the file?

Reply:
Blank lines were removed.

Reply:
@geeknoid these blank lines wer eintroduced by me.  The linter, when run locally from the docker container complain about missing blank lines around blocks.
I mentioned this on slack without context.   Here is the context :)